packages/svelte/src/internal/client/render.js

```diff
@@ -1402,12 +1402,13 @@ function handle_event_propagation(handler_element, event) {
 	const handled_at = event.__root;
 	if (handled_at) {
 		const at_idx = path.indexOf(handled_at);
-		if (at_idx !== -1 && handler_element === document) {
-			// This is the fallback document listener but the event was already handled
-			// -> ignore, but set handle_at to document so that we're resetting the event
+		// @ts-ignore
+		if (at_idx !== -1 && (handler_element === document || handler_element === window)) {
```

Comment:
It just occurred to me that we should probably be using the `ownerDocument` from the target element because of iframes rather than `document`.

Reply:
On first look I thought "yeah makes sense", but on second look: The fallback event listener is set to `document`, too - should that one be using `container_target.ownerDocument`, too? Either way, this sounds like something we can talk about separately.